brayns/engine/Model.h

```diff
@@ -185,7 +185,8 @@ class ModelDescriptor : public ModelParams
         if (_onRemovedCallback)
             _onRemovedCallback(*this);
     }
-
+    void markForRemoval() { _markedForRemoval = true; }
+    bool isMarkedForRemoval() const { return _markedForRemoval; }
```

Comment:
that's protected again :)

Reply:
Impossible, since it is called from the scene.